.svglintrc.js

```diff
@@ -163,6 +164,87 @@ module.exports = {
               }
             }
           },
+          function(reporter, $, ast) {
+            reporter.name = "icon-path";
```

Comment:
I would change this name by... `ineffective-segments` maybe? Seems to me that is too general :thinking:

Reply:
Sure!